### PR TITLE
Main menu and Song Constants

### PR DESCRIPTION
**Pull request by PikalaxALT**

### CODE REVIEW

File: src/help_system_812B1E0.c [hunk 1]
```diff
@@ -1005,16 +1005,16 @@ bool8 sub_812B45C(void)
     return TRUE;
 }
 
-void sub_812B478(void)
+void HelpSystem_Disable(void)
 {
-    gUnknown_3005ECC = 0;
+    gHelpSystemEnabled = 0;
```
Comment (jiangzhengwenjz) [inline src/help_system_812B1E0.c]: FALSE

File: src/help_system_812B1E0.c [hunk 2]
```diff
@@ -1005,16 +1005,16 @@ bool8 sub_812B45C(void)
...
 }
 
-void sub_812B484(void)
+void HelpSystem_Enable(void)
 {
     if (gUnknown_203ADFA != 2 && gUnknown_203ADFA != 3)
     {
-        gUnknown_3005ECC = 1;
+        gHelpSystemEnabled = 1;
```
Comment (jiangzhengwenjz) [inline src/help_system_812B1E0.c]: TRUE

File: src/quest_log.c [hunk 3]
```diff
@@ -1603,7 +1603,7 @@ void sub_81120AC(u8 taskId)
             gTextFlags.autoScroll = FALSE;
             gUnknown_2036E28 = 0;
             sub_8082740(0);
-            gUnknown_3005ECC = 1;
+            gHelpSystemEnabled = 1;
```
Comment (jiangzhengwenjz) [inline src/quest_log.c]: TRUE

File: src/link.c [hunk 4]
```diff
@@ -1577,7 +1577,7 @@ static void CB2_PrintErrorMessage(void)
         {
             if (JOY_NEW(A_BUTTON))
             {
-                sub_812B484();
+                HelpSystem_Enable();
                 PlaySE(SE_PIN);
                 gWirelessCommType = 0;
```
Comment (jiangzhengwenjz) [inline src/link.c]: FALSE

File: src/main_menu.c [hunk 5]
```diff
@@ -0,0 +1,795 @@
...
+    if (!gPaletteFade.active)
+    {
+        SetGpuReg(REG_OFFSET_WIN0H, 0);
+        SetGpuReg(REG_OFFSET_WIN0V, 0);
+        SetGpuReg(REG_OFFSET_WININ, 0x0001);
+        SetGpuReg(REG_OFFSET_WINOUT, 0x0021);
+        SetGpuReg(REG_OFFSET_BLDCNT, BLDCNT_TGT1_BG0 | BLDCNT_TGT1_BG1 | BLDCNT_TGT1_BG2 | BLDCNT_TGT1_BG3 | BLDCNT_TGT1_OBJ | BLDCNT_TGT1_BD | BLDCNT_EFFECT_DARKEN);
+        SetGpuReg(REG_OFFSET_BLDALPHA, BLDALPHA_BLEND(0, 0));
+        SetGpuReg(REG_OFFSET_BLDY, 7);
+        if (gTasks[taskId].tMenuType == 0)
```
Comment (jiangzhengwenjz) [inline src/main_menu.c]: MAIN_MENU_NEWGAME

File: src/link.c (hunk 4 above)
Comment (PikalaxALT) [reply src/link.c]: actually no this is not a bool

**Pull request merged**